docs/TypeScript.md

```diff
@@ -35,6 +35,9 @@ This example will get you up and running with Fastify and TypeScript. It results
     }
   }
   ```
+
+*Note: Set `target` property in `tsconfig.json` to `esnext` to avoid [FastifyDeprecation](https://github.com/fastify/fastify/issues/3284) warning.*
```

Comment:
```suggestion
*Note: Set `target` property in `tsconfig.json` to `es2017` or greater to avoid [FastifyDeprecation](https://github.com/fastify/fastify/issues/3284) warning.*
```